src/Utils/MiscUI/HighColorTab.hpp

```diff
@@ -45,7 +45,7 @@ namespace HighColorTab
 			return std::unique_ptr<CImageList>();
 		}
 
-		if( 0 == apILNew->Create( 16, 16, ILC_COLOR32|ILC_MASK, 0, 1 ) )
+		if( 0 == apILNew->Create( w, h, ILC_COLOR32|ILC_MASK, 0, 1 ) )

```

Comment:
Adjusted space. Please check if it'd be alright in this way. https://github.com/TortoiseGit/TortoiseGit/pull/205/commits/3d8b61439ee4f2b5e4323a24d013d91f2f0f7618#diff-40ba2ca12f830d9b4547f8973a8d4b93